_rules/text-contrast-afw4f7.md

```diff
@@ -189,6 +189,36 @@ This text is using the default user agent text color and background color. By de
 <div role="button">My button!</div>
 ```
 
+#### Passed Example 12
```

Comment:
I think some of the new examples should be in Korean or Chinese instead of Japanese. Not blocking, though.

Reply:
@Jym77, 
I've made some revisions to the examples by substituting Japanese with Chinese, utilizing the Chinese rendition of the WCAG 2.1 title.

Regarding Korean, it appears that only "Hanja" (Chinese characters) are affected by this rule. However, given my unqualified Korean language knowledge, I'm unable to craft a meaningful sentence using these Hanja characters. Any assistance in providing a valid Korean sentence would be greatly appreciated.

Reply:
Korean Hangul might be another special case - lines are laid out in 2 dimensions unlike Chinese and Japanese - it's unclear what point size and line height mean here.

> For example, the Korean word for "honeybee" (kkulbeol) is written as 꿀벌, not ㄲㅜㄹㅂㅓㄹ.
https://en.wikipedia.org/wiki/Hangul

Reply:
It looks like Korean Hanja is very rare - its use and teaching was banned in 1968 - so we can probably ignore it. The ban has been repealed recently, but as a result most Korean people don't know Hanja:
https://en.wikipedia.org/wiki/Hanja#Decline_of_Hanja

The Hangul syllable blocks mentioned in the [comment above](https://github.com/act-rules/act-rules.github.io/pull/2121#discussion_r1346372231) are actually precomposed into single Unicode code points, so point size is the size of the precomposed glyph.
https://www.w3.org/TR/klreq/#hangulcodes

This is similar to the way accents like circumflex and vowels are precomposed in French, as in hôpital